app/pages/project/workflow-selection.jsx

```diff
@@ -60,17 +52,11 @@ class WorkflowSelection extends React.Component {
     ) {
       selectedWorkflowID = workflowFromURL;
       activeFilter = false;
```

Comment:
The active filter isn't being used by Gravity Spy, only by Notes from Nature. Is that correct?

Reply:
The active filter is true for most projects' workflows, but changed to false if the user is an admin, project owner, tester, or expert that may have a reason to load an inactive workflow. `this.checkUserRoles(project, user)` this returns true for these user roles and then the active filter is changed to false.

For Notes from Nature, the next set of conditions is for them and projects like them. 
```
else if (workflowFromURL &&
      project.experimental_tools &&
      project.experimental_tools.indexOf('allow workflow query') > -1
    ) {
      selectedWorkflowID = workflowFromURL;
```

The active filter isn't changed and stays true.

Reply:
I forgot about previewing workflows in development. 😳 

I remember we talked about workflow assignment for inactive workflows on #5171 where you mentioned owners being able to load inactive workflows, so thought I'd check because the workflow selection code does not allow that.
https://github.com/zooniverse/Panoptes-Front-End/pull/5171#pullrequestreview-189074992

Reply:
Easy enough to change though, by setting the filter from the user roles earlier in the code.
`let activeFilter = this.checkUserRoles(project, user);`